markdown/en/api/ff-asn.api.md

```diff
@@ -35,38 +35,38 @@ ___
-| **hideHiddenLinksContainer(animated?)** | Close the hiddenLink container. Parameter animated:boolean is optional. |
-| **toggleHiddenLinksContainer(animated?)** |Toggle the hiddenLink container. Parameter animated:boolean is optional. |
+| **toggle(animate?: boolean)** | Toggles the asn-group |
+| **show(animate?: boolean)** | Opens the asn-group |
+| **hide(animate?: boolean)** | Closes the asn-group |
+| **showDetailedLinksContainer(animate?: boolean)** | Opens the detailedLink container. Parameter `animate: boolean` is optional. |
+| **hideDetailedLinksContainer(animate?: boolean)** | Close the detailedLink container. Parameter `animate: boolean` is optional. |
+| **toggleDetailedLinksContainer(animate?: boolean)** | Toggle the detailedLink container. Parameter `animate: boolean` is optional. |
+| **showHiddenLinksContainer(animate?: boolean)** | Opens the hiddenLink container. Parameter `animate: boolean` is optional. |
+| **hideHiddenLinksContainer(animate?: boolean)** | Close the hiddenLink container. Parameter `animate: boolean` is optional. |
+| **toggleHiddenLinksContainer(animate?: boolean)** |Toggle the hiddenLink container. Parameter `animate: boolean` is optional. |
 
 ## `ff-asn-group-element`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-| **selected**&nbsp;(String) **Options**:&nbsp;true,&nbsp;false (default: empty) | The alignment of the After Search Navigation. |
+| **selected**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: empty) | The alignment of the After Search Navigation. |
 | **element**&nbsp;(Object) (default: empty) | The filter data. |
 | **group**&nbsp;(Object) (default: empty) | The filter group data. |
 
 ### Methods
 | Name | Description |
 | ---- | ----------- |
-| **clone** | Enables cloning of elements with all properties, behaviors, private fields, staes and HTML templates. |
+| **clone** | Return a deep copy of the contained elements including all properties, behaviors, private fields, states and HTML templates. |
 
 ## `ff-asn-group-slider`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
 | **opened**&nbsp;(Boolean) | Determines if the filter group is expanded. If you want to have a group opened by default: `<ff-asn-group-slider opened>` does the trick. |
-| **collapsible**&nbsp;(String) **Options**:&nbsp;true,&nbsp;false (default: true) | Determines, if the filter group is collapsible. |
-| **for-group**&nbsp;(String) | Determines, which filter group the template should be applied to. If all is selected, then the template is applied to all groups if no other template suits the requirements. |
+| **collapsible**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "true") | Determines, if the filter group is collapsible. |
+| **for-group**&nbsp;(String) | Determines, which filter group the template should be applied to. If `all` is selected, then the template is applied to all groups if no other template suits the requirements. |
```

Comment:
like in ff-asn-group I dislike ~~If all is selected~ ->  _If its value is not set or is "all"_

Reply:
I disagree, it should be:
If set to `"all"`, ...
The default should be described by `(default: "all")` in the first column.